likes_lookup.go

```diff
@@ -0,0 +1,54 @@
+package gotwtr
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+)
+
+func likesLookUpUsers(ctx context.Context, c *client, id string, opt ...*LikesLookUpByTweetOpts) (*LikesLookUpByTweetResponse, error) {
+	// check id
+	if len(id) == 0 {
+		return nil, errors.New("likes look up by tweet: id parameter is required")
+	}
+	LikesLookUpByTweet := timeline + "?id=" + id + "/tweets"
```

Comment:
This api endpoint is `https://api.twitter.com/2/tweets/:id/liking_users`.
But this line's endpoint is `https://api.twitter.com/2/users?id={id}/tweets`
(timeline = "https://api.twitter.com/2/users")

you may mistake implementation.